### PR TITLE
[4.0] Internet Explorer message

### PR DESCRIPTION
**Pull request by brianteeman**
Although you can kind of use the admin with IE it is not supported and will not work 100%. This PR uses css to detect if you are using ie10 or 11 and displays a warning if you are. (hopefully this will save us from numerous reports that something doesn't work)

To test `npm i` or `node build.js --compile.css`

Make sure that you do not see a message with your normal browser
If you have access to IE10 or 11 then make sure you do see a message

### Expected result on IE
image

### CODE REVIEW

File: administrator/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -642,6 +642,7 @@ JGLOBAL_VOTES="Votes"
 JGLOBAL_VOTES_ASC="Votes ascending"
 JGLOBAL_VOTES_DESC="Votes descending"
 JGLOBAL_WARNCOOKIES="Warning! Cookies must be enabled to access the Administrator Backend."
+JGLOBAL_WARNIE="Warning! Internet Explorer ahould not be used for proper operation of the Administrator Backend."
```
Comment (C-Lodder) [inline administrator/language/en-GB/joomla.ini]: `ahould` >> `should`

File: administrator/templates/atum/scss/blocks/_login.scss [hunk 2]
```diff
@@ -247,3 +247,13 @@ label {
     color: theme-color("danger");
   }
 }
+
+@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/blocks/_login.scss:256]: I think this media query should go after line 275 else the `display:none` will take presidence, as it comes after. Something like this should be ok:

```
.ie11 {
  display: none;

  @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
    display: block;
  }
}
```

Comment (brianteeman) [reply administrator/templates/atum/scss/blocks/_login.scss:256]: As it is working as it is right now I dont see the need for a change.

Comment (C-Lodder) [reply administrator/templates/atum/scss/blocks/_login.scss:256]: Because using `!important` is bad practice

Comment (Fedik) [reply administrator/templates/atum/scss/blocks/_login.scss:256]: with suggested changes you will be no need to use `!important` (as it currently)

Comment (brianteeman) [reply administrator/templates/atum/scss/blocks/_login.scss:256]: will change it and hopefully someone will review all the gazillion of uses of !important elsewhere

File: administrator/language/en-GB/joomla.ini [hunk 3]
```diff
@@ -642,6 +642,7 @@ JGLOBAL_VOTES="Votes"
 JGLOBAL_VOTES_ASC="Votes ascending"
 JGLOBAL_VOTES_DESC="Votes descending"
 JGLOBAL_WARNCOOKIES="Warning! Cookies must be enabled to access the Administrator Backend."
+JGLOBAL_WARNIE="Warning! Internet Explorershould not be used for proper operation of the Administrator Backend."
```
Comment (ciar4n) [inline administrator/language/en-GB/joomla.ini]: Missing space.. `Explorershould`

Comment (brianteeman) [reply administrator/language/en-GB/joomla.ini]: thanks - fixed

File: administrator/templates/atum/scss/blocks/_login.scss (hunk 2 above)
Comment (ciar4n) [reply administrator/templates/atum/scss/blocks/_login.scss:256]: > will change it and hopefully someone will review all the gazillion of uses of !important elsewhere

Just having a look at atum and you are indeed correct. There seems to be a general reliance on using Bootstraps utility classes, all of which use `!important`. The template then overrides these utility classes (for media queries and rtl) with another `!important`

File: administrator/templates/atum/scss/blocks/_login.scss [hunk 4]
```diff
@@ -247,3 +247,14 @@ label {
...
 }
+
+ /* Detect IE 10+ and display alert */
+.ie11 {
+  display: none;
+}
+
+@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+  .ie11 {
+      display: block;
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_login.scss]:
```suggestion
    display: block;
```

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on 2c82c3d45e966e4e1285c5873e4ae8d33f5d59a8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27879">issues.joomla.org/tracker/joomla-cms/27879</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b2bd0eaa2b8c875a9f00e75b0d81a96f5b828997<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27879">issues.joomla.org/tracker/joomla-cms/27879</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27879">issues.joomla.org/tracker/joomla-cms/27879</a>.</sub>

**Comment by jwaisner** [comment]
Setting back to pending. Missing 1 more test before moving to commit changes.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27879">issues.joomla.org/tracker/joomla-cms/27879</a>.</sub>

**Comment by Quy** [comment]
It is only coding style change that does not affect test results.

**Comment by jwaisner** [comment]
@Quy @brianteeman That detail needs to be called out in testing. The notes do not indicate this and will only cause confusion and problems for Testers.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27879">issues.joomla.org/tracker/joomla-cms/27879</a>.</sub>

**Comment by ciar4n** [comment]
I have tested this item :white_check_mark: successfully on b2bd0eaa2b8c875a9f00e75b0d81a96f5b828997<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27879">issues.joomla.org/tracker/joomla-cms/27879</a>.</sub>

**Comment by brianteeman** [comment]
thanks

**Pull request merged**